netutil.py

```diff
@@ -236,15 +240,9 @@ def valid_vlan(vlan):
     return True
 
 def valid_ip_addr(addr):
-    if not re.match('^\d+\.\d+\.\d+\.\d+$', addr):
-        return False
-    els = addr.split('.')
-    if len(els) != 4:
-        return False
-    for el in els:
-        if int(el) > 255:
-            return False
-    return True
+    ipv4_re = '^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}'
```

Comment:
Why are we not using the ipaddress library here, is it just because we don't include it in the install image?

Reply:
The code doesn't use `ipaddress` because it long predates its introduction in Python 3.3.

We could use one of the unofficial Python 2 backports but I'd probably prefer that we move to Python 3 first and then do further improvements.

In the meantime, I think using a regex here would be OK if the repeated patterns were condensed.

Reply:
I removed the use of regex to instead use `socket.inet_pton`. Is that okay with you?